addons/im_livechat/static/src/embed/common/chat_window_patch.js

```diff
@@ -27,6 +27,7 @@ patch(ChatWindow.prototype, {
             this.props.chatWindow.show({ notifyState: this.thread?.state !== "open" });
         } else {
             await super.close();
+            this.thread?.delete();
```

Comment:
Hello, thank you for pointing the issue. Could you add a test?

Also, this is deleting non-temporary thread, which is what the commit tried to prevent.
session_sate might become "not persisted" even for actual thread (after closing feedback panel I think).
Can we limit to delete real temporary thread?

Reply:
The thread becomes "not persisted" after the feedback panel is displayed. So, I think It's ok to delete that channel as you have already left the channel. And this LOC would run only when you click on "Close" on feedback panel.

Reply:
But the channel still exists after closing it, there's conceptually no reason that it should be deleted.
We no longer display it in current flow but that's a soft spec.

Reply:
But, what use is that channel after you have closed it?

Reply:
The idea is to have as much common code in livechat/portal/backend as possible.

In the base code a closed chat window would still display in messaging menu, discuss sidebar, etc.

Eventually, we'd even like to have a common store shared between all open tabs to avoid fetching the same data many times at each page load.

Deleting a channel that still exists goes against all of those goals.